sql/expression_value.cc

```diff
@@ -1767,6 +1767,13 @@ embedding(Date ts,
     return result;
 }
 
+StorageType
+ExpressionValue::
+getEmbeddingType() const
+{
+    return embedding_->storageType_;
```

Comment:
Too dangerous: this will segfault if it's not an embedding.  We should also make it work for a non-embedding type, although that can happen later.